00_Base/src/interfaces/messages/index.ts

```diff
@@ -33,6 +34,14 @@ export enum EventGroup {
   Transactions = 'transactions',
 }
 
+export const eventGroupFromString = (source: string): EventGroup => {
```

Comment:
Why is this in this PR? There are enough changes need just to move to using an npm workspace without adding additional features. Please reduce the changes per PR where possible.

Reply:
@thanaParis I am trying to make a lot happen in a small period of time here, if needed I can walk you through the PR in detail and explain these, but if you feel that breaking these out into separate PRs, as it would require additional effort, then I am open to a discussion. The changes I am making tend to have a good reason and all of the changes are open for discussion. I can break these out into multiple PRs but in this case, this was needed as EventGroup is being parsed from the process.env which was the setup prior to this PR